onyx.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<manifest>
+    <project name="CyanogenMod/android_device_oppo_common" path="device/oppo/common" remote="github" revision="ec34d17cd81afd459071e2f6e663b1d65338257e" />
```

Comment:
Is it there yet? I can't find it, but yes I had local changes in it because some packages are in the device tree and in there, so i needed to remove them from android_device_oppo_common.